aip/0123.md

```diff
@@ -51,18 +51,31 @@ Examples of resource types include:
 
 ### Annotating resource types
 
-APIs **should** annotate the resource types for each resource in the API:
+APIs **should** annotate the resource types for each resource in the API using
+the [`google.api.resource`][resource] annotation:
 
 ```proto
+// A representation of a Pub/Sub topic.
 message Topic {
   option (google.api.resource) = {
     type: "pubsub.googleapis.com/Topic"
   };
+
+  // The name of the topic.
+  // Format: projects/{project}/topics/{topic}
```

Comment:
Don't we want to include the format in an annotation as well? Or is that coming later?

Reply:
Good point. No clue why I did not do that.